Mathlib/Data/Set/Intervals/OrdConnected.lean

```diff
@@ -0,0 +1,276 @@
+/-
+Copyright (c) 2020 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury G. Kudryashov
+
+! This file was ported from Lean 3 source module data.set.intervals.ord_connected
+! leanprover-community/mathlib commit ba2245edf0c8bb155f1569fd9b9492a9b384cde6
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Data.Set.Intervals.UnorderedInterval
+import Mathlib.Data.Set.Lattice
+
+/-!
+# Order-connected sets
+
+We say that a set `s : set α` is `OrdConnected` if for all `x y ∈ s` it includes the
+interval `[[x, y]]`. If `α` is a `DenselyOrdered` `ConditionallyCompleteLinearOrder` with
+the `OrderTopology`, then this condition is equivalent to `IsPreconnected s`. If `α` is a
+`LinearOrderedField`, then this condition is also equivalent to `Convex α s`.
+
+In this file we prove that intersection of a family of `OrdConnected` sets is `OrdConnected` and
+that all standard intervals are `OrdConnected`.
+-/
+
+-- porting note: namespace `Interval` is not found, commented out following line
+-- open Interval
+
+open OrderDual (toDual ofDual)
+
+namespace Set
+
+section Preorder
+
+variable {α β : Type _} [Preorder α] [Preorder β] {s t : Set α}
+
+/-- We say that a set `s : set α` is `OrdConnected` if for all `x y ∈ s` it includes the
+interval `[[x, y]]`. If `α` is a `DenselyOrdered` `ConditionallyCompleteLinearOrder` with
+the `OrderTopology`, then this condition is equivalent to `IsPreconnected s`. If `α` is a
+`LinearOrderedField`, then this condition is also equivalent to `Convex α s`. -/
+class OrdConnected (s : Set α) : Prop where
+  -- porting note: added docstring
```

Comment:
```suggestion
```
I don't think this is necessary, it happens everywhere and it is already documented in the wiki.